### ISSUE TITLE
RPM generation : missing signature

### ISSUE THREAD

**Issue by bouriquet** [bluebanquise/bluebanquise#151]
Hi, 

RPMs builded by BB under https://bluebanquise.com/repository/ should be signed and a RPM containing your GPG key should be provide ?

**Comment by actatux** [comment]
This could be a great feature for a next release.

**Comment by oxedions** [comment]
Agreed. 1.3 is expected to focus on user interface and security. This could be part of the security side.

**Comment by oxedions** [comment]
This issue need to be moved to bluebanquise infrastructure repository.
